### DISCUSSION TITLE
Contabilidade na  Versão 16

### DISCUSSION THREAD

**Discussion by gilsonlima10** [OCA/l10n-brazil#3483] [General]
A Versão 16 esta sem as opções de contabilidade, foram retiradas ou ainda não foram migradas da versão 14 oficial? existe previsão para a versão 16 se tornar a default?

**Comment by rvalyi** [comment]
Ola @gilsonlima10 se vc olhar no Runboat da versão 16.0 [aqui](https://runboat.odoo-community.org/webui/builds.html?repo=OCA/l10n-brazil&target_branch=16.0) vc vai encontrar sim as opçoes de contabilidade. A Odoo SA sempre deixa mais escondido os menus de contabilidade e tira funcionalidades do Odoo CE, mas com esse nosso modulo vc encontra os menus de volta: https://github.com/OCA/account-financial-tools/tree/16.0/account_usability . E tem outros módulos e repos que se tornam necessários para fazer a contabilidade de forma open source na v16 como esse por exemplo https://github.com/OCA/account-reconcile/

Sobre tornar a v16 a branch default. Isso não é o que deve decidir a sua escolha de versão. Tem muitos projetos sendo tocados na v16, varios outros em prod na v12 ou até ainda na v8. Escolher uma versão ou outra depende das suas necessidades e capacidades para completar o resolver os problemas para a seu determinado caso de uso. Isso vale para qualquer repo da OCA. Algumas empresas vivem de iludir as pessoas passando a impressão que tudo vai ser facil, que basta comprar um pacote mágico com eles para tornar tudo fácil, mas isso não passa de golpes que se aproveitam da falta de informação sobre o Odoo a e OCA aqui no Brasil.

Um dos motivos que a gente ainda não fez a troca da branch 16.0 eh que tem um contribuidor que errou no procedimento técnico da migração do modulo l10n_br_base [1] e ferrou os commits nas branches 15.0 e 16.0; ele perdeu uns 500 commits. A longo prazo isso ferra o uso de ferramentas como oca-port etc e vamos ter que concertar essa cagada dele antes de poder recomendar a v16.0 para pessoas mais leigas. Pois isso vai envolver fazer um git push --force, o que vai forçar as pessoas a re-fazer os clones ou as branches que pretendem mesclar nas v15 ou v16. Mas mesmo assim, esse force push vai mudar apenas o histórico git da branch 16.0 e quase não vai tocar no conteúdo, ou seja quem sabe o que faz já pode usar a branch 16.0.

Uma outra coisa um pouco impactante que é urgente acertar antes de recomendar a v16 para pessoas mais iniciantes é essa mudança no modelo de dados mais básico: https://github.com/OCA/l10n-brazil/pull/3386


[1] https://github.com/OCA/l10n-brazil/pull/2164

**Comment by gilsonlima10** [comment]
resolvido..ativei o modulo sugerido e funcionou!

**Comment by rvalyi** [comment]
quem tiver interesse pode acompanhar o processo de reconstrução da branch 16.0 do OCA/l10-brazil [aqui](https://github.com/OCA/l10n-brazil/issues/3490)  Em breve eu acho que vamos resolver esse problema importante.
